content/docs/latest/howtocommit_27362108.md

```diff
@@ -56,12 +56,11 @@ If a commit introduces new test failures, the preferred process is to revert the
 When you commit/merge a Pull Request, please:
 
 1. Ensure that the Pull Request has a +1 vote, and that **24 hours have elapsed since the first +1 vote was cast** on the Pull Request. Note that this rule appears in the Hive Bylaws. Do not ignore it.
-2. Include the Jira issue id in the commit message, along with a short description of the change and the name of the contributor. Be sure to get the issue id right, as this causes Jira to link to the change in Git/Github.   
-
-	1. if contributor is you then add the following suffix to commit message "(<you>, reviewed by <reviewer>)". Example: "HIVE-123. Add awesomesauce to the optimizer. (jvs, reviewed by Ashutosh Chauhan)"
-	2. if contributor is not you then add the following suffix to commit message "(<contributor> via <you>)". Example: "HIVE-123. Add awesomesauce to the optimizer. (Mike Brakestoner, reviewed by Ashutosh Chauhan)"
-		1. Additionally: "Co-authored-by: Ayush Saxena <ayushsaxena@[apache.org](http://apache.org)>" can be used to attribute any additional code contributors.
-3. Resolve the issue as fixed, thanking the contributor and the reviewers. **Always set the "Fix Version"** at this point, but please only set a single fix version, the earliest release in which the change will appear. However, if a patch is backported to a point release (such as 1.0.2) then multiple fix versions should be set so that the automated release notes can list the Jira issue for the point release as well as the primary release.
+2. Include the Jira issue id in the commit message, along with a short description of the change. Be sure to get the issue id right, in order to have proper links between Jira and GitHub.
+3. Append the PR id, in the commit subject to track the relation between the commit and the respective PR.
```

Comment:
Should we give an commit example here?

Reply:
By the way, **PR** can be replaced by **Pull Request** to avoid possible ambiguity caused by abbreviation.

Reply:
The PR id means "(#5758)" of "HIVE-28782: Ensure Hive constructs URI's with IPv6 literals safely. (#5758)"? We may delete the sentence, assuming we don't intentionally remove it.

Reply:
Fixed in https://github.com/apache/hive-site/pull/47/commits/1735ec1c9bf5541770b434c7d787b848e6eb1cd9